src/map/location.hpp

```diff
@@ -178,6 +178,13 @@ struct map_location {
 	int x, y;
 };
 
+class map_location_hasher {
+public:
+	size_t operator()(const map_location& m) const {
+		return m.y * 1000 + m.x;
```

Comment:
Is there an advantage of this over the other map location hasher defined at the bottom of this header?

Reply:
I removed the duplicate map_location hasher.